src/components/calcite-tile-select/calcite-tile-select.tsx

```diff
@@ -67,6 +67,27 @@ export class CalciteTileSelect {
 
   private input: HTMLCalciteCheckboxElement | HTMLCalciteRadioButtonElement;
 
+  //--------------------------------------------------------------------------
+  //
+  //  Private Methods
+  //
+  //--------------------------------------------------------------------------
+
+  private checkFirstTileSelect(): void {
+    const tileSelects = document.querySelectorAll(`calcite-tile-select[name=${this.name}]`);
+    let firstCheckedTileSelect: HTMLCalciteTileSelectElement;
+    if (tileSelects && tileSelects.length > 0) {
+      tileSelects.forEach((tileSelect: HTMLCalciteTileSelectElement) => {
+        if (firstCheckedTileSelect) {
+          tileSelect.checked = false;
+        } else if (tileSelect.checked) {
+          firstCheckedTileSelect = tileSelect;
+        }
+        return tileSelect;
```

Comment:
Does this need to return an element? could it just return?

Reply:
good catch, I'll update this same line in radio-button.